pkgs/games/openra/build-engine.nix

```
@@ -0,0 +1,69 @@
+{ lib, buildDotnetModule, dotnetCorePackages
+, SDL2, freetype, openal, lua51Packages
+}:
+engine:
+
+buildDotnetModule rec {
+  pname = "openra";
+  version = "${engine.name}-${engine.version}";
```

```suggestion
  pname = "openra-${engine.pname}";
  inherit (engine) version;
```

that probably breaks the code that uses pname below but I think the name of the engine belongs more to the package name than its versin